roles/node/tasks/001-health-check.yml

```diff
@@ -21,7 +21,7 @@
 
   - name: Health check | Run health check
     ansible.builtin.uri:
-      url: "http://127.0.0.1:{{ node_rpc_port }}"
+      url: "http://127.0.0.1:{{ (node_parachain_relay_chain_rpc_urls | length > 0) | ternary(node_parachain_rpc_port, node_rpc_port) }}"
```

Comment:
If default value is `node_parachain_relay_chain_rpc_urls: [""]` this condition is always  `True`: 
```
- hosts: localhost
  gather_facts: true
  vars:
    list_e: []
    list_f0: [""]
    list_f1: ["aaa"]
    list_f2: ["aaa","bbb"]
  tasks:
   - debug:
       msg: "length list_e: {{ list_e | length }}, list_f0: {{ list_f0 | length }}, list_f1: {{ list_f1 | length }}, list_f2: {{ list_f2 | length }}"
```
output:
```
"msg": "length list_e: 0, list_f0: 1, list_f1: 1, list_f2: 2"
```